src/compiler/scala/tools/nsc/classpath/ZipAndJarFileLookupFactory.scala

```diff
@@ -212,10 +213,9 @@ final class FileBasedCache[K, T] {
     e.cancelTimer()
 
     new Closeable {
-      var closed = false
+      val closed = new AtomicBoolean(false)
```

Comment:
There could be an Abide rule that false is the default. (Still fun to rag on Abide.)